src/nav/style-guide.yml

```diff
@@ -74,11 +76,19 @@ pages:
         path: /docs/style-guide/processes-procedures/publication-checklist
       - title: Embed videos
         path: /docs/style-guide/processes-procedures/embed-videos
+      - title: Update left navigation pane
```

Comment:
Similar teeny nitpick: This is called `left navigation pane` in sidebar but `left-navigation pane` in doc title. The no hyphen version is more correct IMO